SeLU function.py

```diff
@@ -0,0 +1,7 @@
+import ivy
+
+def SELU(x, lambdaa = 1.0507, alpha = 1.6732):
+    if x >= 0:
+        return ivy.array(int(lambdaa * x))
+    else:
+        return ivy.array(int(lambdaa * alpha * (ivy.exp(x) - 1)))
```

Comment:
Hi @SaimR036, I'm assuming you are trying to implement `selu` as a frontend function. Please go through our [contribution guide](https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#open-tasks). You should position the function in an existing frontend file instead of adding a new script, and you should also add a frontend test. Thanks!